src/components/layer-metadata/layer-metadata.vue

```diff
@@ -61,10 +79,36 @@ function closeLayerMetadata() {
         class="col-span-3 grid gap-2 grid-cols-3"
       >
         <span class="font-bold">{{ t('Description') }}</span>
-        <span
-          v-dompurify-html="layerMetadata.description"
-          class="col-span-2"
-        ></span>
+        <span class="col-span-2">
+          <span v-dompurify-html="description"></span>
+          <button
+            :title="
+              t('Display full description', {
+                ns: 'client',
+              })
+            "
+            v-if="!displayFullDescription"
+            @click="showFullDescription"
+            class="text-secondary hover:underline"
+          >
+            ...
+          </button>
+          <button
+            :title="
+              t('Hide full description', {
+                ns: 'client',
+              })
+            "
+            v-if="
+              displayFullDescription &&
+              (description?.length || '') > MAX_DESCRIPTION_LENGTH
+            "
+            @click="hideFullDescription"
+            class="text-secondary hover:underline"
+          >
+            -
+          </button>
+        </span>
       </div>
       <layer-metadata-item
         v-if="layerMetadata.legalConstraints"
```

Comment:
Missing css styling for `<h4>{{ t('Legend') }}</h4>`

Reply:
Also the title 'avec légende' is visible even if there is not legend in production.

Reply:
Thought it'd be more logic to not display "avec légende" if there is none. Didn't like the style on production, but you're right I adapted the font size accordingly.

Reply:
I also do not like the style in production. + agree with you, it is more logic to hide the title when there is no legend.